src/cogent3/app/composable.py

```diff
@@ -704,7 +699,7 @@ def _as_completed(self, dstore, parallel=False, par_kw=None, **kwargs) -> Genera
 
 def is_composable(obj):
```

Comment:
I think for most uses of is_compsable, is_app is sufficient.  

But one test of concat specificly is testing an app that is not composable, so I added a function `is_app_composable()`

Reply:
I also added an optional boolean argument to `def apps(force: bool = False):` which forces a reload of the ExtensionManager, which is otherwise lazy loaded and cached.